store/sqlstore/user_store.go

```diff
@@ -1064,8 +1076,9 @@ func (us SqlUserStore) Count(options model.UserCountOptions) store.StoreChannel
 		}
 
 		if options.TeamId != "" {
-			query = query.LeftJoin("TeamMembers ON Users.Id = TeamMembers.UserId").Where("TeamMembers.TeamId = ? AND TeamMembers.DeleteAt = 0", options.TeamId)
+			query = query.LeftJoin("TeamMembers AS tm ON u.Id = tm.UserId").Where("tm.TeamId = ? AND tm.DeleteAt = 0", options.TeamId)
 		}
+		query = applyViewRestrictionsFilter(query, options.ViewRestrictions, false)
```

Comment:
This is more a question than a change request. In this case, we can have two left joins for `TeamMembers` if `options.TeamId != ""`, is this ok for both PostgreSQL and MySQL?

Reply:
Added a test and looks like it works well :)